tests/OSPSuite.Core.IntegrationTests/ModuleIntegrationTests.cs

```diff
@@ -53,6 +53,15 @@ public void should_have_added_a_tumor_organ_in_heart_organ()
          tumor_pls.ShouldNotBeNull();
       }
 
+      [Observation]
+      public void should_have_left_the_original_container_of_the_spatial_structure_intact()
+      {
+        var module2= _simulationConfiguration.ModuleConfigurations[1].Module;
+        var lungModule2 = module2.SpatialStructure.FindByName(Lung);
+        lungModule2.ShouldNotBeNull();
+        lungModule2.ParentContainer.ShouldBeNull();
```

Comment:
before the fix, this was set to the orgnaism of the first spatial structure (what @PavelBal found)
Now it's left intact